lib/assets/connection_cell/main.js

```diff
@@ -108,7 +104,6 @@ export function init(ctx, info) {
         :value="modelValue"
         @input="$emit('update:data', $event.target.value)"
         v-bind="$attrs"
-        :required="required"
```

Comment:
We still need this on the input though?

Reply:
not anymore, but I removed it before your review I guess 😝

Reply:
But it will be added to div and input elements